Auth0/CredentialsManager.swift

```diff
@@ -142,12 +142,12 @@ public struct CredentialsManager {
     ///
     /// - Parameters:
     ///   - scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during original Auth.
-    ///   - minTTL: minimum time in milliseconds the access token must remain valid to avoid being renewed.
+    ///   - minTTL: minimum time in seconds the access token must remain valid to avoid being renewed.
     ///   - callback: callback with the user's credentials or the cause of the error.
     /// - Important: This method only works for a refresh token obtained after auth with OAuth 2.0 API Authorization.
     /// - Note: [Auth0 Refresh Tokens Docs](https://auth0.com/docs/tokens/concepts/refresh-tokens)
     #if WEB_AUTH_PLATFORM
-    public func credentials(withScope scope: String? = nil, minTTL: Int = 0, callback: @escaping (CredentialsManagerError?, Credentials?) -> Void) {
+    public func credentials(withScope scope: String? = nil, minTTL: Float = 0, callback: @escaping (CredentialsManagerError?, Credentials?) -> Void) {
```

Comment:
What is the purpose of the float as we're using seconds?

Reply:
Since seconds are a lot less granular than milliseconds, floats are needed to allow fractions of seconds, instead of whole seconds only.

Reply:
What would be the use case that would require milliseconds? Let's look at a use case from the original issue of requiring a token with 10 minutes left e.g. 600 seconds.

Reply:
You're right, I'll change it back to Int.

Reply:
Changed in https://github.com/auth0/Auth0.swift/pull/399/commits/b02ee119124e4b78d2339314ded5421517b3a987.